nerves_defconfig

```diff
@@ -1,6 +1,8 @@
 BR2_arm=y
 BR2_arm1176jzf_s=y
 BR2_TAR_OPTIONS="--no-same-owner"
+BR2_PRIMARY_SITE="http://sources.buildroot.net"
```

Comment:
```suggestion
```

Removing this instructs Buildroot to go to the package's source location. I assume that the Buildroot people will want this so we don't hammer their site.